packages/admin-panel/src/layout/Breadcrumbs.jsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { ArrowBack } from '@material-ui/icons';
+import { Typography, Breadcrumbs as MuiBreadcrumbs, IconButton } from '@material-ui/core';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid ${({ theme }) => theme.palette.grey['400']};
+`;
+
+const BackButton = styled(IconButton)`
+  color: ${({ theme }) => theme.palette.text.primary};
+  margin-right: 0.5rem;
+`;
+
+const ActiveBreadcrumb = styled(Typography)`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+export const Breadcrumbs = ({ parent, displayProperty, title, details, getDisplayValue }) => {
+  const itemDisplayValue = getDisplayValue ? getDisplayValue(details) : details?.[displayProperty];
+
+  return (
+    <Wrapper>
+      <BackButton component={Link} to={parent?.to || '/'}>
+        <ArrowBack />
+      </BackButton>
+      <MuiBreadcrumbs separator="|">
+        <ActiveBreadcrumb component={Link} to={parent?.to || '/'}>
+          {parent?.title}
+        </ActiveBreadcrumb>
```

Comment:
The back button and the parent button seem to always be taking me back to the `Surveys` page and tab?

Reply:
This should be fixed in the other PR